packages/turbo-types/src/types/config-v1.ts

```diff
@@ -0,0 +1,301 @@
+   * @defaultValue `false`
+   */
+  signature?: boolean;
+
+  /**
+   * Indicates if the remote cache is enabled. When `false`, Turborepo will disable
+   * all remote cache operations, even if the repo has a valid token. If true, remote caching
+   * is enabled, but still requires the user to login and link their repo to a remote cache.
+   * Documentation: https://turbo.build/repo/docs/core-concepts/remote-caching
+   *
+   * @defaultValue `true`
+   */
+  enabled?: boolean;
+}
+
+export interface RootSchemaV1 extends BaseSchemaV1 {
+  /**
+   * A list of globs to include in the set of implicit global hash dependencies.
+   *
+   * The contents of these files will be included in the global hashing
+   * algorithm and affect the hashes of all tasks.
+   *
+   * This is useful for busting the cache based on:
+   *
+   * - .env files (not in Git)
+   *
+   * - any root level file that impacts package tasks
+   * that are not represented in the traditional dependency graph
+   * (e.g. a root tsconfig.json, jest.config.js, .eslintrc, etc.)
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#globaldependencies
+   *
+   * @defaultValue `[]`
+   */
+  globalDependencies?: Array<string>;
+
+  /**
+   * A list of environment variables for implicit global hash dependencies.
+   *
+   * The variables included in this list will affect all task hashes.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#globalenv
+   *
+   * @defaultValue `[]`
+   */
+  globalEnv?: Array<EnvWildcardV1>;
+
+  /**
+   * An allowlist of environment variables that should be made to all tasks, but
+   * should not contribute to the task's cache key, e.g. `AWS_SECRET_KEY`.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#globalpassthroughenv
+   *
+   * @defaultValue `null`
+   */
+  globalPassThroughEnv?: null | Array<EnvWildcardV1>;
+
+  /**
+   * A priority-ordered (most-significant to least-significant) array of project-anchored
+   * Unix-style paths to `.env` files to include in the global hash.
+   *
+   * @defaultValue `null`
+   */
+  globalDotEnv?: null | Array<AnchoredUnixPath>;
+
+  /**
+   * Configuration options that control how turbo interfaces with the remote cache.
+   *
+   * Documentation: https://turbo.build/repo/docs/core-concepts/remote-caching
+   *
+   * @defaultValue `{}`
+   */
+  remoteCache?: RemoteCacheV1;
+
+  /**
+   * Enable use of the UI for `turbo`.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#ui
+   *
+   * @defaultValue `"stream"`
+   */
+  ui?: UIV1;
+
+  /**
+   * Disable check for `packageManager` in root `package.json`
+   *
+   * This is highly discouraged as it leaves `turbo` dependent on system
+   * configuration to infer the correct package manager.
+   *
+   * Some turbo features are disabled if this is set to true.
+   *
+   * @defaultValue `false`
+   */
+  dangerouslyDisablePackageManagerCheck?: boolean;
```

Comment:
This field wasn't present in v1
```suggestion
```

Reply:
updated in faa5d8120c4faf1a9e839edc8bf8fd9b473d2829